composer.json

```diff
@@ -22,7 +22,8 @@
         "twig/twig":                "~1.12"
     },
     "require-dev": {
-      "symfony/expression-language": "~2.4"
+      "symfony/expression-language": "~2.4",
+      "fabpot/php-cs-fixer": "~1.0"
```

Comment:
hmm, but what if the CI runs with version X and locally with version Y, this could result in different results...